### PR TITLE
EOS-14103: Update cortx-prereqs.sh for centos7.8 support

### PR DESCRIPTION
**Pull request by SomwanshiAnjali**

### CODE REVIEW

File: cli/src/cortx-prereqs.sh [hunk 1]
```diff
@@ -251,16 +251,21 @@ fi
 
 echo -n "INFO: Checking if kernel version is correct.........................." 2>&1 | tee -a ${LOG_FILE}
 
+grep -q "Red Hat" /etc/*-release && {
```
Comment (ipoddubnyy) [inline cli/src/cortx-prereqs.sh:254]: does CentOS also return "Red Hat"?
and is it "Red Hat" or "RedHat"?

**Review by ipoddubnyy — APPROVED** [review]
approved (providing my question is answered)

File: cli/src/cortx-prereqs.sh (hunk 1 above)
Comment (SomwanshiAnjali) [reply cli/src/cortx-prereqs.sh:254]: 1. On centos above grep command will return nothing and hence kernel version will be set to 3.10.0-1127.el7.x86_64
2. In /etc/os-release and /etc/redhat-release the release name  is mentioned as "Red Hat Enterprise Linux Server"

**Comment by 83bhp** [comment]
The changes look ok to me. But as discussed in the morning please add following changes:
1. make -t argument compulsory for deployment on CentOS, if not provided error out and exit.
2. Use the OS repos from Seagate hosted mirrors. This will lock the versions for packages installed from OS repos, remember the standard repos keeps changing and unavailability of one mirror makes yum to try the next one in the list, this ends up consuming more time. So internal hosted mirrors are required.

**Comment by SomwanshiAnjali** [comment]
> The changes look ok to me. But as discussed in the morning please add following changes:
> 
> 1. make -t argument compulsory for deployment on CentOS, if not provided error out and exit.
> 2. Use the OS repos from Seagate hosted mirrors. This will lock the versions for packages installed from OS repos, remember the standard repos keeps changing and unavailability of one mirror makes yum to try the next one in the list, this ends up consuming more time. So internal hosted mirrors are required.



> The changes look ok to me. But as discussed in the morning please add following changes:
> 
> 1. make -t argument compulsory for deployment on CentOS, if not provided error out and exit.
> 2. Use the OS repos from Seagate hosted mirrors. This will lock the versions for packages installed from OS repos, remember the standard repos keeps changing and unavailability of one mirror makes yum to try the next one in the list, this ends up consuming more time. So internal hosted mirrors are required.


@83bhp 
> first point I have addressed with latest changes but instead of making -t compulsory can we update default value of 
url_local_repo_commons="http://cortx-storage.colo.seagate.com/releases/cortx/uploads/centos/centos-7.7.1908/" to
http://cortx-storage.colo.seagate.com/releases/cortx/uploads/centos/centos-7.8.2003/

because if we consider deployment using mounted iso we won't have this build url (base url of hosted iso)

> For second I believe we provisioner do not add/update CentOS based repos.

**Review by andkononykhin2 — APPROVED** [review]

**Review by ypise — APPROVED** [review]

**Review by ipoddubnyy — APPROVED** [review]
looks ok to me

**Pull request merged**